phoenix-ios/phoenix-ios/views/SendView.swift

```diff
@@ -691,16 +697,23 @@ struct ValidateView: View, ViewName {
 			
 			content
 			
-			if mvi.model is Scan.Model_LnurlPayFlow_LnUrlPayFetch {
+			if mvi.model is Scan.Model_LnurlPayFlow_LnurlPayFetch {
 				LnurlFetchNotice(
 					title: NSLocalizedString("Fetching Invoice", comment: "Progress title"),
 					onCancel: { didCancelLnurlPayFetch() }
 				)
+			} else if mvi.model is Scan.Model_LnurlWithdrawFlow_LnurlWithdrawFetch {
+				LnurlFetchNotice(
+					title: NSLocalizedString("Sending Invoice", comment: "Progress title"),
```

Comment:
`Sending Invoice` could be confusing since the user is withdrawing funds. How about `Forwarding invoice`?

Reply:
Done in da9166e